Taylor and Francis Group.js

```diff
@@ -0,0 +1,287 @@
+{
+	"translatorID": "71f28f90-764a-42a8-8107-d87252da9b50",
+	"label": "Taylor and Francis Group",
```

Comment:
"Taylor & Francis eBooks"? I think "Group" is just the company, and "eBooks" is the relevant product here. It's an ampersand regardless.

Reply:
Lack of ampersand is to match [the other T&F translator](https://github.com/zotero/translators/blob/master/Taylor%20and%20Francis%2BNEJM.js), which also calls it "Taylor and Francis." Is it more important to be consistent or correct? Either way, you're right, "eBooks" would be better than "Group."

Reply:
Correct is better.

Reply:
Done.